config/initializers/secrets.rb

```diff
@@ -2,6 +2,9 @@
 
 # Generates a few 'secrets' meant to represent API keys or something similar
 
+require 'vulneruby_engine/application_record'
+require 'vulneruby_engine/secret'
```

Comment:
Explicit requires added due to the following deprecation warning now being issued regarding autoloading during initialisation.

```
DEPRECATION WARNING: Initialization autoloaded the constants VulnerubyEngine::ApplicationRecord, VulnerubyEngine::Secret.

Being able to do this is deprecated. Autoloading during initialization is going
to be an error condition in future versions of Rails.
```